src/renderer/tvgInitializer.cpp

```diff
@@ -91,19 +91,22 @@ static bool _buildVersionInfo()
 Result Initializer::init(CanvasEngine engine, uint32_t threads) noexcept
 {
     auto nonSupport = true;
+    if (!(engine & CanvasEngine::Sw) && !(engine & CanvasEngine::Gl)) {
```

Comment:
if ((int) engine == 0) return InvalidArguments;

Reply:
I updated it.